Sources/PackageLoading/ManifestLoader.swift

```diff
@@ -371,7 +371,7 @@ public final class ManifestLoader: ManifestLoaderProtocol {
             // Check that the product references existing targets.
             for target in product.targets {
                 if !manifest.targetMap.keys.contains(target) {
-                    try diagnostics.emit(.productTargetNotFound(productName: product.name, targetName: target))
+                    try diagnostics.emit(.productTargetNotFound(productName: product.name, targetName: target, validTargets: manifest.targetMap.keys.sorted()))
```

Comment:
We have a couple of places in which we sort targets by name.  This is fine for this PR but at some point I think it would make sense to show them in the order in which they're declared in the manifest, since that presumably makes some logical sense for package authors.  I could be that the internal structures have discarded that order and will need to keep it somehow (perhaps through an OrderedDictionary).